cmd/kpod/run.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/libpod"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var runDescription = "Runs a command in a new container from the given image"
+
+var runCommand = cli.Command{
+	Name:        "run",
+	Usage:       "run a command in a new container",
+	Description: runDescription,
+	Flags:       createFlags,
+	Action:      runCmd,
+	ArgsUsage:   "IMAGE [COMMAND [ARG...]]",
+}
+
+func runCmd(c *cli.Context) error {
+	if err := validateFlags(c, createFlags); err != nil {
+		return err
+	}
+	runtime, err := libpod.NewRuntime()
```

Comment:
Why are we using NewRuntime here instead of getRuntime? We lose the ability to get storage configuration from flag/config file

Reply:
fixed